OpenSim/Common/Component.cpp

```diff
@@ -369,8 +359,26 @@ void Component::baseAddToSystem(SimTK::MultibodySystem& system) const
 
 void Component::componentsAddToSystem(SimTK::MultibodySystem& system) const
 {
-    for (const auto& compRef : _orderedSubcomponents) {
-        compRef->addToSystem(system);
+    // If _orderedSubcomponents is specified, then use this Component's
+    // specification for the order in which subcomponents are added. At a
+    // minimum the order for all immediate subcomponents must be specified.
```

Comment:
The doxygen for `setNextSubcomponentInSystem()` should mention this requirement as well.